presto-native-execution/presto_cpp/main/http/HttpClient.cpp

```diff
@@ -448,7 +451,10 @@ folly::SemiFuture<proxygen::HTTPTransaction*> HttpClient::createTransaction(
   }
   auto idleSessionFuture = idleSessions_->getIdleSession();
   auto getFromIdleSession =
-      [self = shared_from_this(), this, handler](
+      [self = shared_from_this(),
+       this,
+       handler,
+       startTimeMs = velox::getCurrentTimeMs()](
```

Comment:
Do we suspect that execution of `getFromIdleSession` takes long time?

Reply:
Yes, we already see some delays with a custom build in shadow